cmd/kubeadm/app/phases/upgrade/postupgrade.go

```diff
@@ -41,6 +46,9 @@ import (
 	"k8s.io/kubernetes/pkg/util/version"
 )
 
+var v190alpha3 = version.MustParseSemantic("v1.9.0-alpha.3")
```

Comment:
This var is not used. Should be removed?

Reply:
It's still used (in line 147: https://github.com/kubernetes/kubernetes/pull/60359/files#diff-668202917ffe6a202af0eea0d7bcf87aR147) and necessary to keep it for now. It can be removed in 1.11 cycle.